nbdev/frontmatter.py

```diff
@@ -36,9 +40,13 @@ def _md2dict(s:str):
     return res
 
 # %% ../nbs/API/frontmatter.ipynb 6
+def dict2fm(d): return f'---\n{yaml.dump(d)}\n---\n\n'
```

Comment:
These seem to be mainly for internal use, so they should have names starting with `_`.

(If they're also for external use, they should have a docstring and docments)

Reply:
done